internal/provisioner/fluentbit.go

```diff
@@ -80,12 +79,38 @@ func (f *fluentbit) NewHelmDeployment(logger log.FieldLogger) *helmDeployment {
 	if err != nil {
 		logger.WithError(err).Error("unable to lookup private zone name")
 	}
+	domainNameSplitted := strings.Split(privateDomainName, ".")
+	sumoLogic := `[OUTPUT]
+	Name  forward
+	Match  *
+	Host  example.placeholder.mattermost.com
```

Comment:
I need to discuss with @DSchalla if it is ok to have hardcoded the proper URL which is internal

Reply:
We may want to template that out as well into an env var or something so we can update it easily in the future.

Reply:
Yes, fully agree what @gabrieljackson said.